include/gripper/core/gripper_info.hpp

```diff
@@ -90,7 +90,8 @@ class GripperInfoMessageHandler : eeprom::accessor::ReadListener {
     void visit(const InstrumentInfoRequest &m) {
         // Start a serial number read. Respond with CAN message when read
         // completes.
-        // serial_number_accessor.start_read();
+
+#if PCBA_PRIMARY_REVISION == 'b'
```

Comment:
Is there an "a" revision of the gripper?

Reply:
we have a proto board but I don't think any of them exist anymore